src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -66,7 +66,7 @@
     public static final String EXCEPTION_MSG = "general.exception";
 
     /** Logger for Checker. */
-    private static final Log LOG = LogFactory.getLog(Checker.class);
+    private final Log log;
```

Comment:
@samuel-gu ,
keep it static, Main is not part of checkstyle as library so, static usage is completely ok in it.